scripts/check_pyo3_link_script.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# This script is only used in CI to check if the pyo3 backend is linked correctly
+echo $(pwd)
```

Comment:
question: Why is the output path needed here? If output is required, provide more information.